app/src/containers/ListOfNews/util.js

```diff
@@ -0,0 +1,7 @@
+exports.compareDate = void 0;
+
+var compareDate = function compareDate(firstDate, secondDate) {
```

Comment:
`var` is legacy. Checkout `let` and `const`

You don't have to write name after `function` when you are assigning the function to a variable, the variable name becomes the function name.